app/javascript/article-form/components/Help/index.jsx

```diff
@@ -96,10 +93,15 @@ export const Help = ({ previewShowing, helpFor, helpPosition, version }) => {
   );
 };
 
+Help.defaultProps = {
+  helpFor: '',
+  helpPosition: '',
```

Comment:
should this be a number?

Reply:
Yes, because it was throwing error on string